skimage/transform/_thin_plate_splines.py

```diff
@@ -0,0 +1,297 @@
+import numpy as np
+import scipy as sp
+
+
+class TPSTransform:
```

Comment:
I think `TpsTransform` would follow our naming convention for classes more closely.